### ISSUE TITLE
PDF does not have style from theme

### ISSUE THREAD

**Issue by jeffreywolf** [travs/markdown-pdf#136]
From the README I expected that CSS (from the theme) would be applied to the markdown resulting in style and formatting similar to that seen using Markdown Preview, but when I convert the Markdown file to a PDF the PDF does not have any of the style from the theme.   

I installed markdown-pdf through Settings > Packages. I used ctrl+shift+C to convert the Markdown file in the active window.

I am using Atom version 1.12.2 and markdown-pdf 1.5.0.

**Comment by tehmul** [comment]
Yup, same here - Atom version 1.12.2 and markdown-pdf 1.5.0 on Linux.

**Comment by BlueHatbRit** [comment]
I've found the plugin works best when the atom theme and syntax theme are both set to "Atom Dark". Then it mirrors the pdf viewer pretty well! Or are you talking about custom css in styles.less?

**Comment by dpo** [comment]
Changing the theme/syntax doesn't do it for me. The PDF uses an obnoxiously large font size. How do you change that?

**Comment by BlueHatbRit** [comment]
@dpo, markdown-pdf styles feed from the markdown-preview panes output HTML (found using `ctrl+shift+m`). To edit those styles you have two options, the first is changing the syntax theme which will change _some_ aspects of the final pdf.

The main way to change the style of the final pdf is using some custom css. If you open the "stylesheet" for by going to the Atom menu (i'm not sure where this is on windows, probably under _File_) you can then open the "Stylesheet".

![atom-settings](https://cloud.githubusercontent.com/assets/1068477/21857628/00119a66-d81e-11e6-89d2-1fecc2af2e8b.png)

That stylesheet uses less which is very similar to css. Because markdown-pdf uses the html output from markdown-preview you need to edit the css for the markdown-preview element. Here's an example of a full stylesheet for changing the text size which will result in a smaller text size on the final pdf.

```less
.markdown-preview.markdown-preview {
  font-size: 11px;

  // Table
  table {
    th {
      font-size: 11px;
      text-align: center;
    }
    td {
      font-size: 11px;
    }
  }
}
```

As a bonus I've also provided the code for changing tables as well because I pulled this directly from my stylesheet.

## Tldr:

1. Open up the atom stylesheet and throw the above code in there and save it.
2. Go back to your markdown file and generate it again.
3. Profit.

**Comment by travs** [comment]
Have updated the package to no longer support atom's editor styling (i.e. the theme) by default.
The advice just above about including styles still applies however.
